### PR TITLE
Solve Puzzle FIve

### PR DESCRIPTION
**Pull request by Waseb-untwale**
# Maine puzzle Five Solve kar liya
#6
- #6

### CODE REVIEW

File: puzzle-five.md [hunk 1]
```diff
@@ -19,13 +19,12 @@ Spotting patterns is important in writing good programs.
...
-- The first light House Contains the value ___. Because ____________
+- The first light House Contains the value **36**. Because **each number in increases by 5**
 
-- The second light House Contains the value ___. Because ____________
+- The second light House Contains the value **28**. Because **each number is increses in +1**
 
-- The third light House Contains the value ___. Because ____________
-
-- The last light House Contains the value ___. Because ____________
+- The third light House Contains the value **64**. Because **every number multiply it's own value**
```
Comment (s-m-quadri) [inline puzzle-five.md:26]: Add a blank line between question 3 and 4.

**Review by s-m-quadri — CHANGES_REQUESTED** [review]
Please correct changes

File: puzzle-five.md (hunk 1 above)
Comment (Waseb-untwale) [reply puzzle-five.md:26]: okk

**Review by s-m-quadri — APPROVED** [review]
Good.

**Comment by s-m-quadri** [comment]
This is not puzzle two, make changes. in description.

**Comment by s-m-quadri** [comment]
Done. Congratulations 🌹

**Pull request merged**